### PR TITLE
Added py-spy and changed the stress script to use it

### PR DESCRIPTION
**Pull request by hackaugusto**
Adds py-spy as a dev dependency and change the stress test to use it. This should be a more performant version since it is effectively a sampler profiler, while our current implementation is really a tracer.

### CODE REVIEW

File: requirements/requirements-ci.txt [hunk 1]
```diff
@@ -77,7 +77,6 @@ jinja2==2.10.1
 jsonschema==3.0.1
 lazy-object-proxy==1.4.1
 lru-dict==1.1.6
-macholib==1.14            # via pyinstaller
```
Comment (hackaugusto) [inline requirements/requirements-ci.txt]: I don't understand why this dependency changed, @ulope any ideas?

Comment (ulope) [reply requirements/requirements-ci.txt]: It's a mac only dependency of pyinstaller. pip-tools is unfortunately not good at dealing with plattform specific deps.

Just leave it in.

**Comment by karlb** [comment]
I liked how easy it was to enable profiling in any raiden instance with the old approach. With this approach we only get it for the stress test. Any reason for doing it differently, this time?

**Comment by hackaugusto** [comment]
> Any reason for doing it differently, this time? Is it because you lack the nursery in the main raiden code?

You can get profiling information with py-spy just as easily:

https://github.com/benfred/py-spy#record

**Pull request merged**